docs/navigation-structure.md

```diff
@@ -237,6 +237,42 @@ Currently, the navigation structure is limited to 3 levels: grandchild pages can
 
 ---
 
+## Top-Level Grouping for Pages in Sidebar Nav
+
+By default, when using Parent and Children pages, the Parent page will show in the sidebar nav folded, hiding the Child pages from users of your site. Additionally, the Parent page will exist as a page on it's own, which isn't always desired. To have Parent-style group headings for organizing pages better, you can use the [Document Collections]({% link docs/configuration.md %}#document-collections) feature.
+
+Creating collections instead of parent pages, and setting `nav_fold` and `nav_exclude` to false in your `_config.yml` will show the Parent page's name but it will not be a blank page with Child pages, just a sidebar heading.
+
+### Example Setup:
+{: .no_toc }
+
+```yaml
+_config.yml:
+  collections:
+    collection-one:
+      permalink: "/:collection/:path/"
+      output: true
+    collection-two:
+      permalink: "/:collection/:path/"
+      output: true
+  just_the_docs:
+    collections:
+      collection-one:
+        name: Collection One
+        nav_fold: false
+        nav_exclude: false
+      collection-two:
+        name: Collection Two
+        nav_fold: false
+        nav_exclude: false
+
+In your site directory:
+  _collection-one/PAGES-GO-HERE
+  _collection-two/PAGES-GO-HERE
+```
+
+---
+
```

Comment:
````suggestion
## Grouping pages with collections

Pages can also be grouped together by using Jekyll's and Just the Docs's [collections]({% link docs/configuration.md %}#document-collections) feature. In contrast to using [pages with children](#pages-with-children), pages grouped by collection are grouped by a shared header (the name of the collection) instead of a page.

The `nav_fold` configuration option works for collection-grouped pages. For more information, please refer to the [collections documentation]({% link docs/configuration.md %}#document-collections).

### Example (grouping by collection)
{: .no_toc }

The following example sets up two Jekyll collections, `collection-one` and `collection-two`:

- any document placed within `_collection-1/` will be grouped under the `Collection One` header by default. Since `nav_fold` is set to `true`, the pages will be folded by default.
- any document placed within `_collection-1/` will be grouped under the `Collection Two` header by default. Since `nav_fold` is set to `false`, the pages will be expanded by default.

```yaml
_config.yml:
  collections:
    collection-one:
      permalink: "/:collection/:path/"
      output: true
    collection-two:
      permalink: "/:collection/:path/"
      output: true
  just_the_docs:
    collections:
      collection-one:
        name: Collection One
        nav_fold: true
      collection-two:
        name: Collection Two
        nav_fold: false
```
---

````

Reply:
That is definitely more concise and much cleaner. I totally didn't think about an example of the folded option, that's a good catch! I think your changes keep the idea clear while maintaining brevity and consistency. I still have lots to learn about keeping documentation in that manner!

Reply:
کیسا دست میں نہیں سمجھا کچھ ہم بھی طور پر ویڈیو یا تصویر دکھائے